README.md

```diff
@@ -811,6 +812,28 @@ will generate html like this:
 <option value="2">London</option>
 ```
 
+### Formatters
+
+#### currency
+Format the currency value according to country.
+
+Parameters:
+```
+value [number] The numerical value of currency. (Required)
+args [arbitrary arguments] The currency formatting parameters. (Optional)
+```
+
+Returns: `string`
+
+Usage:
+```
+{{currency 1000000 code='USD'}}  => $1,000,000
+{{currency 1000000 code='USD' precision=2}}  => $1,000,000.00
+{{currency 1000000 code='EUR' precision=2}}  => 1 000 000,00 €
```

Comment:
For europian currency, the commonly used format is 1 000 000,00 €. But can be changed using `thousand="," decimal="."` parameters.

References: 
- http://ux.stackexchange.com/questions/9105/international-currency-formatting-guidelines-currency-codes
- https://www.kvraudio.com/forum/viewtopic.php?t=219595